testfiles/issue67/repeatInRepeat.h

```diff
@@ -23,11 +23,11 @@
 #ifndef __ASSEMBLER__
 struct repeatInRepeat {
   /* [0x0]: REPEAT (comment missing) */
-  struct repA {
+  struct {
     /* [0x0]: BLOCK (comment missing) */
-    struct repA_block1 {
+    struct {
       /* [0x0]: REPEAT (comment missing) */
-      struct repA_block1_repB {
```

Comment:
I just saw that there is actually a proper prefixing of `repeat` and `block` structures. But it does not append the top level name (the one of the memory map, in this example `repeatInRepeat`). Depending on how you see things (and if you think that these substructs are actually used to), one might also simply add the top level name as an additional prefix:

```c
struct repA_block1_repB {
```

would become 

```
struct repeatInRepeat_repA_block1_repB {
```
instead of how it is done in this PR:

```
struct {
```